migrate/repo_checks.py

```diff
@@ -54,6 +75,97 @@ def dry_run(self):
         raise NotImplementedError
 
 
+class RequireTeamPermission(Check):
+    """
+    A generic check that is not meant to be used directly
+    but is used to reduce some copy-pasta.
+    """
```

Comment:
```suggestion
    """
    Require that a team has a certain level of access to a repository.

    To use this class as a check, create a subclass that specifies a particular
    team and permission level, such as RequireTriageTeamAccess below.
    """
```
The current docstring states your opinion on what this class is for, but it doesn't tell the reader what the class actually does or how they should use it.